src/App.tsx

```diff
@@ -10,32 +11,21 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0]);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
+        { ` ${selectedTabId.title}` }
```

Comment:
use the mentor's example, you could just use interpolation for the whole titile
<img width="779" alt="image" src="https://user-images.githubusercontent.com/102411130/233004238-55767bb7-336d-418f-8211-096f71836942.png">